member.go

```diff
@@ -22,11 +22,14 @@ import (
 )
 
 type Member struct {
-	client                   *Client
-	Id                       string   `json:"id"`
-	AvatarHash               string   `json:"avatarHash"`
-	Bio                      string   `json:"bio"`
-	BioData                  string   `json:"bioData"`
+	client     *Client
+	Id         string `json:"id"`
+	AvatarHash string `json:"avatarHash"`
+	Bio        string `json:"bio"`
+	BioData    struct {
+		Emoji struct {
+		} `json:"emoji"`
+	} `json:"bioData"`
```

Comment:
The formatting is broken here.

Can you use "gofmt" command to fix it? https://blog.golang.org/go-fmt-your-code

My personal favorite cmd to fix the whole repository including imports is:
`goimports -w $(find . -type f -name '*.go' -not -path "./vendor/*")`